eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -125,8 +128,16 @@ jobs:
           BuildArgs: "$(BuildArgs)"
           Env: "$(CmakeEnvArg)"
 
-      - script: ctest -C Debug -V --tests-regex ${{ parameters.CtestRegex }} --no-compress-output -T Test
+      - pwsh: |
+          ctest `
+            -C Debug `
+            -V `
+            --tests-regex '${{ parameters.CtestRegex }}' `
+            --exclude-regex '${{ parameters.CtestExcludeRegex }}' `
+            --no-compress-output `
+            -T Test
         workingDirectory: build
+        displayName: Test
```

Comment:
Good catch.

Fixes https://github.com/Azure/azure-sdk-for-cpp/issues/3082 I think